actions/md-confluence-sync/mark-sync.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# $USER is your smartcontract email
+# $PASSWORD is the api token created on the https://id.atlassian.com/manage-profile/security/api-tokens page
+# Mermaid diagrams are rendered locally and inlined into the HTML artifact as PNGs
+
+mark -f "$FIILES" \
+  --edit-lock \
+  --title-from-h1 \
+  -u "$USER" \
+  -p "$TOKEN" \
+  -b https://smartcontract-it.atlassian.net/wiki/ \
```

Comment:
Maybe this should be an input? This is a public repo, and someone else might want to use it as well.